ast/parser.go

```diff
@@ -898,6 +901,36 @@ func (p *Parser) parseSome() *Expr {
 	return NewExpr(decl).SetLocation(decl.Location)
 }
 
+func (p *Parser) parseEvery() *Expr {
+	qb := &Every{}
+	qb.SetLoc(p.s.Loc())
+
+	p.scan()
+	if term := p.parseTermInfixCall(); term != nil {
```

Comment:
Shall we use early termination to save one identation level which eases readability in IDEs?